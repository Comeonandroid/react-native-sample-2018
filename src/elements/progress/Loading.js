import React, { PureComponent } from 'react'
import styled from 'styled-components'

import { View, Text } from '../index'

class Loading extends PureComponent {
  render() {
    return (
      <Container>
        {/* TODO: change for rotating icon */}
        <Text>Loading</Text>
      </Container>
    )
  }
}

const Container = styled(View) ``

export default Loading
