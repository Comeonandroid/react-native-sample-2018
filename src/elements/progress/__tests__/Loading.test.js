import 'react-native'
import React from 'react'
import { itBehavesLike } from '../../../specs/shared_examples/index'

import Loading from '../Loading'

describe('<Loading />', () => {
  itBehavesLike('aThemedComponent', { subject: <Loading /> })
})
