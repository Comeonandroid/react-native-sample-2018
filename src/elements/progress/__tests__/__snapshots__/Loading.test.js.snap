// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> dark: renders correctly 1`] = `
<View
  style={
    Array [
      Object {},
      Array [
        Object {},
        undefined,
      ],
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#fcfcfc",
          "fontFamily": "Open Sans",
        },
        undefined,
      ]
    }
  >
    Loading
  </Text>
</View>
`;

exports[`<Loading /> light: renders correctly 1`] = `
<View
  style={
    Array [
      Object {},
      Array [
        Object {},
        undefined,
      ],
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#000000",
          "fontFamily": "Open Sans",
        },
        undefined,
      ]
    }
  >
    Loading
  </Text>
</View>
`;
